frontend/src/v5/ui/routes/dashboard/projects/containers/containers.component.tsx

```diff
@@ -72,7 +72,7 @@ export const Containers = (): JSX.Element => {
 						<DashboardListEmptyText>
 							<FormattedMessage
 								id="containers.favourites.emptyMessage"
-								defaultMessage="You haven’t added any Favourites. Click the star on a container to add your first favourite Container."
+								defaultMessage="Click on the star to mark a container as favourite"
```

Comment:
@sanmont3drepo I'm not sure whether you want to update the Spanish translations for these?